config/task.go

```diff
@@ -34,10 +34,10 @@ type TaskConfig struct {
 	// service name in the default namespace.
 	Services []string `mapstructure:"services"`
 
-	// Source is the location the driver uses to fetch dependencies. The source
-	// format is dependent on the driver. For the Terraform driver, the source
-	// is the module path (local or remote).
-	Source *string `mapstructure:"source"`
+	// Module is the path to fetch the Terraform Module (local or remote).
+	// Previously named Source - Deprecated in 0.5
+	Module           *string `mapstructure:"module"`
+	DeprecatedSource *string `mapstructure:"source"`
```

Comment:
I like this rename to deprecated source since it'll be easier to find what to remove in the future!

Reply:
🎉 Can't take credit! I got this pattern from @findkim https://github.com/hashicorp/consul-terraform-sync/pull/140